lib/UI/UI.rb

```diff
@@ -17,6 +17,10 @@ def move_from_user
     move_from_user
   end
 
+  def mode_from_user
```

Comment:
This method name is confusing. I assume it's a `get` method, but I can't really tell since it's not a verb. Furthermore, if it's a Console UI class to interact with users, it's always going to be from the user right, so there would be no need to put it in the method name. Maybe something along the lines of `get_mode` would make more sense?